src/QMCWaveFunctions/tests/test_rotatedspos.cpp

```diff
@@ -0,0 +1,305 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source License.
+// See LICENSE file in top directory for details.
+//
+// Copyright (c) 2016 Jeongnim Kim and QMCPACK developers.
+//
+// File developed by: Mark Dewing, markdewing@gmail.com, University of Illinois at Urbana-Champaign
+//
+// File created by: Mark Dewing, markdewing@gmail.com, University of Illinois at Urbana-Champaign
+//////////////////////////////////////////////////////////////////////////////////////
+
+
+#include "catch.hpp"
+
+#include "OhmmsData/Libxml2Doc.h"
+#include "OhmmsPETE/OhmmsMatrix.h"
+#include "Particle/ParticleSet.h"
+#include "Particle/ParticleSetPool.h"
+#include "QMCWaveFunctions/WaveFunctionComponent.h"
+#include "QMCWaveFunctions/EinsplineSetBuilder.h"
+#include "QMCWaveFunctions/RotatedSPOs.h"
+
+#include <stdio.h>
+#include <string>
+#include <limits>
+
+using std::string;
+
+namespace qmcplusplus
+{
+/*
+  JPT 04.01.2022: Adapted from test_einset.cpp
+  Test the spline rotated machinery for SplineR2R (extend to others later).
+*/
+TEST_CASE("RotatedSPOs via SplineR2R", "[wavefunction]")
+{
+  app_log() << "-------------------------------------------------------------" << std::endl;
```

Comment:
Catch2 will provide basically this output based on the testcase name this is just wasting space if the output of the test is printed.

Unit tests should aim for minimal dependencies and you're not aiming to test app_log().

Reply:
Removed.